client/src/pages/SigninContainer.js

```diff
@@ -2,7 +2,8 @@
 
 import React, { useReducer, useEffect } from "react";
 import { Box, Container, Grid, Typography } from "@material-ui/core";
-import { TextField, Button } from "@material-ui/core";
+import { TextField, Button, IconButton } from "@material-ui/core";
```

Comment:
These two import statements are pulling from the same place (`@material-ui/core`).